mysql_servers.json

```diff
@@ -190,7 +190,8 @@
     },
     {
       "name": "event-broker",
-      "script": "../../_scripts/start_after_mysql.sh",
```

Comment:
I had originally copied this from another start, but it doesn't need to wait for mysql so this isn't needed.

Reply:
But it does depend on auth-server which waits for mysql. It was crashing for me when it starts before auth.

Reply:
Ah, good point! That's why I had that there.

Reply:
Reverted.